integration_tests/src/main/python/window_function_test.py

```diff
@@ -58,19 +58,19 @@
     ('c', DecimalGen(precision=8, scale=3, nullable=True))]
 
 _grpkey_byte_with_nulls = [
-    ('a', IntegerGen()),
+    ('a', RepeatSeqGen(int_gen, length=20)),
```

Comment:
Could you have a comment about why making this change ?

Reply:
The rows in partition-by columns should be repeatable, which is  a real-world case.

Reply:
If the rows are always different, let's have an assumption, each group only has 1 row. then all the agg functions will work on only 1 row, which is not a really good test case.